addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -114,6 +116,9 @@ def _get_uom_unece_code(self, line):
             return UOM_TO_UNECE_CODE.get(xmlid[line.product_uom_id.id], 'C62')
         return 'C62'
 
+    def _export_facturx(self, invoice, invoice_data):
+        return self.env['account.edi.xml.cii']._export_invoice(invoice)[0]
```

Comment:
In https://github.com/odoo/odoo/pull/128049/commits/e007f0e048446debd04126509b8b1e34f57d6a94 , I removed the following lines: https://github.com/odoo/odoo/blob/778bb2549e2ac385b52a3c6a3350fc65d283284d/addons/account_edi_ubl_cii/models/account_move_send.py#L135-L143

I deferred the responsability to inject the Factur-x inside the PDF to the edi_model instead of having `account.move.send` doing it itself.

Writing it now, I think I should rename the method into `embed_facturx_in_pdf` or `inject_facturx` or something like this.

What do you think about it? I tried to make account_move_send cleaner but I can reverse if it gets too complex.